be/src/runtime/local_pass_through_buffer.cpp

```diff
@@ -4,33 +4,50 @@
 
 #include "column/chunk.h"
 #include "common/logging.h"
+#include "runtime/current_thread.h"
+#include "runtime/mem_tracker.h"
 
 namespace starrocks {
 
 // channel per [sender_id]
 class PassThroughSenderChannel {
 public:
-    void append_chunk(const vectorized::Chunk* chunk, size_t chunk_size, int32_t driver_sequence) {
-        auto clone = chunk->clone_unique();
-        {
-            std::unique_lock lock(_mutex);
-            _buffer.emplace_back(std::make_pair(std::move(clone), driver_sequence));
-            _bytes.push_back(chunk_size);
+    PassThroughSenderChannel() = default;
+
+    ~PassThroughSenderChannel() {
+        if (_physical_bytes > 0) {
+            CurrentThread::current().mem_consume(_physical_bytes);
         }
     }
 
+    void append_chunk(const vectorized::Chunk* chunk, size_t chunk_size, int32_t driver_sequence) {
+        // Release allocated bytes in current MemTracker, since it would not be released at current MemTracker
```

Comment:
Is this comment right?

Reply:
Yes, since we call the `mem_release` later.